### PR TITLE
Add Yiddish Language

### PR DESCRIPTION
**Pull request by DarthFlip**

### CODE REVIEW

File: src/io/github/sspanak/tt9/languages/definitions/Yiddish.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+public class Yiddish extends Hebrew {
+	public Yiddish() {
+		super();
+
+		locale = new Locale("ji","JI");
+		dictionaryFile = "ji-utf8.csv";
+		hasUpperCase = false;
+		isPunctuationPartOfWords = true;
+
+
```
Comment (sspanak) [inline src/io/github/sspanak/tt9/languages/definitions/Yiddish.java]:
```suggestion
```

These two are already set in the parent constructor, so it shouldn't be necessary to assign them again.

**Comment by sspanak** [comment]
@DarthFlip, could you also provide a link to the source, where you got the wordlist from?

File: src/io/github/sspanak/tt9/languages/definitions/Yiddish.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
+package io.github.sspanak.tt9.languages.definitions;
+
+import java.util.Locale;
+
+public class Yiddish extends Hebrew {
+	public Yiddish() {
+		super();
+
+		locale = new Locale("ji","JI");
+		dictionaryFile = "ji-utf8.csv";
```
Comment (sspanak) [inline src/io/github/sspanak/tt9/languages/definitions/Yiddish.java:10]: @sspanak, copy the word priorities from the Hebrew wordlist before merging.

File: src/io/github/sspanak/tt9/languages/definitions/Yiddish.java (hunk 1 above)
Comment (DarthFlip) [reply src/io/github/sspanak/tt9/languages/definitions/Yiddish.java]: I originally didn't have those in due to your reasoning but when I tried compiling in Android Studio it gave me a library error due to the punctuation so I just added it to make that error go away. If it really doesnt need it then I will take it out.

**Comment by DarthFlip** [comment]
I got the wordlist from this Yiddish T9 app https://www.ivelt.com/forum/viewtopic.php?p=3498106#p3498106 . I contacted the person who made it and he said he made the wordlist and was happy for it to be used/shared publicly.

File: src/io/github/sspanak/tt9/languages/definitions/Yiddish.java (hunk 1 above)
Comment (sspanak) [reply src/io/github/sspanak/tt9/languages/definitions/Yiddish.java]: That's ok, I'll double check and remove them, if needed.

**Comment by sspanak** [comment]
> I got the wordlist from this Yiddish T9 app https://www.ivelt.com/forum/viewtopic.php?p=3498106#p3498106 . I contacted the person who made it and he said he made the wordlist and was happy for it to be used/shared publicly.

Excellent, thank you! 

I can not verify if there are nonsense words or whatnot, because I can't read them, so I trust you they are fine. :slightly_smiling_face:

**Review by sspanak — APPROVED** [review]

**Pull request merged**